R/Study_Assess.R

```diff
@@ -104,5 +106,22 @@ Study_Assess <- function(
     lAssessments <- NULL
   }
 
+  # extract results
+  lAssessments <- purrr::map(lAssessments, function(x) {
+    purrr::discard(x, is.null) %>%
+      purrr::flatten()
+  })
+
+  if (!bQuiet) {
+    purrr::iwalk(lAssessments, function(x, y) {
+
+      if ("message" %in% names(x)) {
+        cli::cli_alert_warning("Workflow {y} did not run. Error message: {x$parent$message}")
```

Comment:
`x$parent$message` doesn't appear to catch the error message emitted on workflow failure.

Reply:
@samussiah - thanks for reviewing! 

I want to test this out in the same way you're looking at it. Did you put that `stopifnot()` somewhere like [here](https://github.com/Gilead-BioStats/gsm/blob/dev/R/AE_Assess.R#L96)?

`purrr::safely()` captures the entire callstack in a nested list, so we could maybe parse some of that out if needed, but want to know what the most straightforward way to resolve the pipeline error issue would be. E.g, if we don't need to print the error to the console and can capture it further downstream, then we can just remove [this part](https://github.com/Gilead-BioStats/gsm/blob/543507289c34dd4824ee23c95e80b41fe67dab1d/R/Study_Assess.R#L115-L123).

Reply:
@mattroumaya I just threw a random (unexpected) error as opposed to an expected error like the ones in `stopifnot`. The console output will display unexpected errors that actually bomb the workflow, so I don't think it's worth the time to parse the call stack as long as the workflow exits gracefully.

Reply:
That said, it would be pretty slick...